grafana/grafana
# devenv: elastic: removed unsupported versions, first step

in grafana9 we do not support elastic versions below 7.10.x, so this PR removes the unsupported devenv datasources and dashboards.

(note: this is only the first step, we will have to adjust the remaining elastic-search datasources too, but this was an easy to create change, so this is the first step)

how to test:
- checkout the code
- run grafana
- check list of datasources, search for `gdev-elastic`, you should only see 4, all called `gdev-elasticsearch-v7-*`
- go to the dashboard-section, you should have a folder named `gdev dashboards`. delete the whole folder
- now stop grafana-server (the go-process), and start it again. this will recreate the dashboards.
- go to dashboard-section, search dashboards by name: "elastic"
- you should get 3 hits in the "gdev dashboards" folder, : `elasticsearch comparison`, `elasticsearch v7`, `elasticsearch v7 filebeat`
